### PR TITLE
Added badges in Readme.md

### PR DESCRIPTION
**Pull request by AshishMhrzn10**
I added hactoberfest, opensource and aquasec website badges in Readme.md to make it a bit beautiful.  Please feel free to make some more suggestions that I can improve on. 

Contributed to issue #8

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<a href="docs/contributing.md" alt="Hacktoberfest"><img src="https://badgen.net/badge/hacktoberfest/friendly/pink" /></a>
```
Comment (nightlark) [inline Readme.md:1]: The link for this badge doesn’t work — probably because they don’t have a contributing.md file in this repository.

Comment (AshishMhrzn10) [reply Readme.md]: I corrected it. Please have a look.

Comment (nightlark) [reply Readme.md]: For some reason the "Open Source" badge seems to have a transparent background and doesn't really show up. I thought it had a colored background before, so it's strange that now it doesn't.

Comment (AshishMhrzn10) [reply Readme.md]: Sorry, I couldn't understand the error. Can you suggest the solution so that I can work on it.

Comment (nightlark) [reply Readme.md]: hard-to-read-text

The 3rd badge is unreadable.

Comment (AshishMhrzn10) [reply Readme.md]: Screenshot from 2020-10-09 08-29-10
 But its working on mine brother.

Comment (nightlark) [reply Readme.md]: Interesting. It’s working with my copy of chrome but not Firefox. So it’s either the svg does something strange, or a bug in firefox.

Comment (AshishMhrzn10) [reply Readme.md]: Any solution??

Comment (nightlark) [reply Readme.md]: It works with Firefox version 53, stops working at least in firefox versions 54-68. The firefox nightly build (not sure about the age... I'm under the impression browserling.com hasn't added new versions in a while) that draws it correctly.

**Comment by itaysk** [comment]
I'm sorry but we don't need any badges in this project

**Pull request closed without merging**